modules/sdk-core/src/bitgo/inscription/iInscription.ts

```diff
@@ -0,0 +1,25 @@
+import { PreparedInscriptionRevealData, TapLeafScript } from '@bitgo/utxo-ord';
+
+export type TxInfoUnspents = {
```

Comment:
No existing type that could be reused instead?

Reply:
@lcovar check if `utxolib.bitgo.WalletUnspent` can be used here